docs/SOPs/data_retraction_SOP.md

```diff
@@ -40,14 +40,35 @@ Please make sure you have the proper permissions or you are paired with someone
 
 ### Set up gsutil to access staging area
 **If you already have gsutil set up, please skip this step**
-Any wrangler/dev with the s3 permissions set up should be able to retrieve the credentials. Please retrieve the credentials with this command:
+
```

Comment:
(Doesn't let me add comments before this line)]

It feels like in the "needed permission" section we need another one for "Access to AWS secrets". Could you write that one @prabh-t ?

Reply:
I noted there that only devs have access to that GCP credential. I would recommend doing the first option to get access to Terra staging bucket for more traceability who did the changes.

Reply:
are you able to run the `aws get-secret-value ...` command from this page from this page?

Reply:
I removed the instructions here just now. I believe it should work for devs which uses the ingest-devops role in AWS. It's better for other users to access it via theirs google accounts instead of using the exporter gcp account.